app.json

```diff
@@ -19,15 +19,19 @@
       "description": "A password seed is used by password hash generator. ",
       "generator": "secret"
     },
+    "MONGO_URI": {
+      "description": "Uri for MongoDB.",
+      "required": true
+    },
+    "REDIS_REJECT_UNAUTHORIZED": "0",
     "MONGOMS_DISABLE_POSTINSTALL": "1"
   },
   "addons": [
-    "mongolab",
-    "redistogo",
+    "heroku-redis",
     {
       "plan": "bonsai",
       "options": {
-        "version": "7.2"
+        "version": "7.10.2"
```

Comment:
This follows 👉 https://docs.bonsai.io/article/139-which-versions-bonsai-supports